File: app/src/main/java/com/fazemeright/chatbotmetcs622/network/models/response/QueryResponseMessage.java
package com.fazemeright.chatbotmetcs622.network.models.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class QueryResponseMessage {

    @SerializedName("data")
    @Expose
    private Data data;

    public Data getData() {
        return data;
    }

    public void setData(Data data) {
        this.data = data;
    }

    public class Data {


        @SerializedName("responseMsg")
        @Expose
        private String responseMsg;

        public String getResponseMsg() {
            return responseMsg;
        }

        public void setResponseMsg(String responseMsg) {
            this.responseMsg = responseMsg;
        }
    }
}

File: app/src/main/java/com/fazemeright/chatbotmetcs622/network/models/response/QueryResponseMessage.kt
package com.fazemeright.chatbotmetcs622.network.models.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class QueryResponseMessage(
        @SerializedName("data")
        @Expose
        var data: Data? = null)

data class Data(
        @SerializedName("responseMsg")
        @Expose
        var responseMsg: String? = null)

